packages/Utils/src/DTK_Core.cpp

```diff
@@ -74,4 +74,10 @@ void finalize()
 
     dtkIsInitialized = false;
 }
+
+// ETI for initialize
+template void initialize<int &, char **&>( int &argc, char **&argv );
```

Comment:
Can you comment on that one?
I just looked it up and found:
`void Kokkos::initialize(int& narg, char* arg[]);`